packages/components/src/components/PressableHighlight/PressableHighlight.tsx

```diff
@@ -9,7 +9,6 @@ import {
 import { useToken } from "../../hooks/useToken";
 
 export type PressableHightlightProps = {
-    highlightColor?: ColorValue;
```

Comment:
What is PressableHighlight? There's no equivalent in EDS. Might want to use the Button component after adding a ghost variant to it